specs/iota-rs-ENGINEERING-SPEC-0000.md

```diff
@@ -173,140 +173,167 @@ A vector of [Message] Object.
 
 ## `get_unspent_address()`
 
-Return a valid unspent address.
+Return a valid unspent public Bech32 encoded address.
 
 ### Parameters
 
-| Field | Required | Default | Type | Definition |
+| Parameter | Required | Default | Type | Definition |
 | - | - | - | - | - |
-| **seed** | ✔ | - | [Seed] | The seed we want to search for. |
-| **path** | ✘ | `m/0'/0'` | [BIP32Path] | The wallet chain BIP32 path we want to search for. |
-| **index** | ✘ | 0 | u32 | Start index of the address. **Default is 0.** |
+| **seed** | ✔ | - | [Seed] | The seed we want to use. |
+| **account_index** | ✘ | 0 | usize | The account index. |
```

Comment:
Same as above, please add an example of where this reflects in the bip path used internally